assets/js/wcmp-admin.js

```diff
@@ -867,3 +848,36 @@ jQuery(function($) {
   }
 });
 
+/**
+ * Object.assign() polyfill.
+ */
+if (typeof Object.assign !== 'function') {
+  // Must be writable: true, enumerable: false, configurable: true
+  Object.defineProperty(Object, 'assign', {
+    value: function assign(target, varArgs) { // .length of function is 2
+      'use strict';
+      if (target === null || target === undefined) {
+        throw new TypeError('Cannot convert undefined or null to object');
+      }
+
+      var to = Object(target);
+
+      for (var index = 1; index < arguments.length; index++) {
+        var nextSource = arguments[index];
+
+        if (nextSource !== null && nextSource !== undefined) {
+          for (var nextKey in nextSource) {
+            // Avoid bugs when hasOwnProperty is shadowed
```

Comment:
Als de code geminified is, zal `//` voor problemen zorgen 

```suggestion
            /* Avoid bugs when hasOwnProperty is shadowed */
```

Reply:
dat klopt